project_issue_reassign/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+The Responsible (``user_id``) field is made read only, and can instead be
```

Comment:
Ah! OK, that particular case could work.
Another relevant advantage for the wizard is that you can use it from the list view to mass update several Tasks. It's useful for a first-line "dispatcher" use case.